main.go

```diff
@@ -345,3 +368,41 @@ func main() {
 		log.Fatal(err)
 	}
 }
+
+func loadConfig() (Config, error) {
+	confDir, confDirerr := os.UserConfigDir()
+	if confDirerr != nil {
+		return Config{}, confDirerr
+	}
+
+	conf := Config{
+		ClusterName: defaultK3sClusterName,
+		Workers:     0,
+	}
+
+	confFilePath := filepath.Join(confDir, "k3d.yaml")
+	if _, err := os.Stat(confFilePath); err == nil {
+		confFile, err := ioutil.ReadFile(confFilePath)
+		if err != nil {
+			return Config{}, err
+		}
+
+		if err := yaml.Unmarshal(confFile, &conf); err != nil {
```

Comment:
What about using UnmarshalStrict rather than Unmarshal

Reply:
Actually, I don't know.
The question is should we make the program fail in case of unknown configuration key? It would be more explicit to exit with an error message rather than silently ignore unknown config fields.
What do you think about that @msaunois-dzr @zibok?